op-tests/test-blspy-g1.txt

```diff
@@ -0,0 +1,120 @@
+; This file was generated by tools/generate-bls-tests.py
+
+bls_g1_add 0xb756ecad886b575d2c95014c20bcd9b52f1f72e3b5fbc657bacf1e72be707f9aaf76a9156a1628b99d873036beecec10 0xa73e3a434a117bed5c0eab24dce24b2e03bc75254b878900eb01f98cf389329cdd7d83cd8033475ff1ec6b2e10abbb09 => 0x932e81cd868b85b0608705ee603f15e9aa087811dae86254e5496008debed51377bc5c19a6ef9db547e3ac3fd7046bdf | 2789534
```

Comment:
We need `bls_g1_add` with no arguments (to check the identity). Fun fact: the identity is an arbitrarily chosen point in g1 (maybe not completely arbitrary, but there's not a LOT of magic to it).

Reply:
I guess we also want some tests with one element.

Reply:
we have this test case (for `point_add`) here: https://github.com/Chia-Network/clvm_rs/blob/main/test-more-ops.txt#L711

Generally I've split the test cases between `test-bls-ops.txt` for *manually* added test cases and the `test-blspy-*.txt` for the generated ones. I suppose most of the G1 and G2 points can't be validated by manual inspection, and should be generated.